docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -14,7 +14,7 @@ You can configure the Connector runtime environment in the following ways:
 
 ## Connecting to Zeebe
 
-In general, the Connector Runtime will respect all properties known to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe).
+In general, the Connector Runtime will respect all properties known to the [Spring Zeebe SDK](/apis-tools/spring-zeebe-sdk/getting-started.md).
```

Comment:
As the runtime hasnt been migrated yet we should not link to it as of now. 

I am not sure whether this reference is making it easier or is rather confusing to people as they would have to check yet another page for detailed information.

Reply:
Let's just delete it then :)

Reply:
@sbuettner is the following still up-to-date? (Further down the page) Is this how you would connect to SaaS?

```bash
ZEEBE_CLIENT_CLOUD_CLUSTER-ID=xxx
ZEEBE_CLIENT_CLOUD_CLIENT-ID=xxx
ZEEBE_CLIENT_CLOUD_CLIENT-SECRET=xxx
ZEEBE_CLIENT_CLOUD_REGION=bru-2
```

Reply:
@akeller AFAIK these have been replaced with:
```
CAMUNDA_CLIENT_CLUSTER-ID=xxx
CAMUNDA_CLIENT_AUTH_CLIENT-ID=
CAMUNDA_CLIENT_AUTH_CLIENT-SECRET=
CAMUNDA_CLIENT_REGION=bru-2
```
which is aligned with the latest [spring-boot-starter-camunda-sdk](https://github.com/camunda/camunda/tree/1bc7942e987e12f16b1ea1a1bab805586bd3f114/clients/spring-boot-starter-camunda-sdk):  https://github.com/camunda/camunda/blob/1bc7942e987e12f16b1ea1a1bab805586bd3f114/clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java#L650

Reply:
@ana-vinogradova-camunda it looks like this change (or deprecation) was made in 8.6.0. Can you confirm?

Reply:
Updated the 8.6 doc.